### PR TITLE
Documentation & ReadMe Cleanup

### PR DESCRIPTION
**Pull request by MyDrift-user**
# Documentation & ReadMe Cleanup

## Type of Change
- [x] Documentation update

## Description
- Removing Stuff from ReadMe (saved and currently being added into the docs page by @Real-MullaC & me)
- Create docs & assets folder & move stuff there (cleanup & preparation for doc files added for page)
- Move Sections in ReadMe to more logical ones
- Added some Icons
- Added link to latest release at download count badge like the version badge so u don't get redirected to a flashlight.

## Testing
Pushed changes & took a look at it. Seems fine to me.

## Impact
Clean ReadMe and Documentation that is not all over the place look good and make life much easier :shipit:

## Checklist
- [x] My code adheres to the coding and style guidelines of the project.
- [x] I have performed a self-review of my own code.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

File: .docs/Overview.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+  - Legacy Windows Panels: Includes access to legacy Windows panels from Windows 7, allowing users to access familiar and powerful tools. These panels include Control Panel for managing system settings, Network Connections for configuring network adapters and connections, Power Panel for adjusting power and sleep settings, Sound Settings for managing audio devices and settings, System Properties for viewing and modifying system information, and User Accounts for managing user profiles and account settings.
+
+
+- Updates:
+  - Default (Out of Box) Settings: Provides the default settings that come with Windows for updates.
+
+  - Security (Recommended) Settings: Offers recommended settings, including a slight delay of feature updates by 2 years and installation of security updates 4 days after release.
+
+  - Disable All Updates (Not Recommended!): Allows users to disable all Windows updates, but it's not recommended due to potential security risks.
```
Comment (og-mrk) [inline .docs/Overview.md:38]:
```suggestion
## Overview

### Install
  - Install Selection: Organize programs by category and facilitate installation by enabling users to select programs and initiate the installation process with a single click.

  - Upgrade All: Upgrade all existing programs to their latest versions, ensuring users have the most up-to-date and feature-rich software.

  - Uninstall Selection: Effortlessly uninstall selected programs, providing users with a streamlined way to remove unwanted software from their system.

  - Get Installed: Retrieve a comprehensive list of installed programs on the system, offering users visibility into the software currently installed on their computer.

  - Import / Export: Enable users to import or export the selection list of programs, allowing them to save their preferred program configurations or share them with others. This feature promotes convenience and flexibility in managing program selections across different systems.

### Tweaks
  - Recommended Selection: Provides pre-defined templates tailored for desktop, laptop, and minimal configurations, allowing users to select recommended settings and optimizations specific to their system type.

  - Essential Tweaks: Offers a collection of essential tweaks aimed at improving system performance, privacy, and resource utilization. These tweaks include creating a system restore point, disabling telemetry, Wi-Fi Sense, setting services to manual, disabling location tracking, and HomeGroup, among others.

  - Advanced Tweaks: Encompasses a range of various advanced power user tweaks to further optimize the system. These tweaks include removing OneDrive and Edge, disabling User Account Control (UAC), notification panel, among others.

  - Toggles: Adds easy to use, one click shortcuts for toggling dark mode, NumLock on startup, file extensions, sticky keys, among others.

  - Additional Tweaks: Introduces various other tweaks such as enabling dark mode, changing DNS settings, adding an Ultimate Performance mode, and creating shortcuts for WinUtil tools. These tweaks provide users with additional customization options to tailor their system to their preferences.

### Config
  - Features: Allows users to easily install various essential components and features to enhance their Windows experience. These features include installing .NET Frameworks, enabling Hyper-V virtualization, enabling legacy media support for Windows Media Player and DirectPlay, enabling NFS (Network File System) for network file sharing, and enabling Windows Subsystem for Linux (WSL) for running Linux applications on Windows.

  - Fixes: Provides a range of helpful fixes to address common issues and improve system stability. This includes setting up autologon for seamless login experiences, resetting Windows updates to resolve update-related problems, performing a system corruption scan to detect and repair corrupted files, and resetting network settings to troubleshoot network connectivity issues.

  - Legacy Windows Panels: Includes access to legacy Windows panels from Windows 7, allowing users to access familiar and powerful tools. These panels include Control Panel for managing system settings, Network Connections for configuring network adapters and connections, Power Panel for adjusting power and sleep settings, Sound Settings for managing audio devices and settings, System Properties for viewing and modifying system information, and User Accounts for managing user profiles and account settings.


### Updates
  - Default (Out of Box) Settings: Provides the default settings that come with Windows for updates.

  - Security (Recommended) Settings: Offers recommended settings, including a slight delay of feature updates by 2 years and installation of security updates 4 days after release.

  - Disable All Updates (Not Recommended!): Allows users to disable all Windows updates, but it's not recommended due to potential security risks.
```
Changing each subsection of this Overview section from a list item to a header will result in a more readable documentation.

Also, thanks for this amazing PR @MyDrift-user 😄
It really improves upon the current documentation of WinUtil.

**Review by og-mrk — APPROVED** [review]
Besides the suggestion left in [this review comment](https://github.com/ChrisTitusTech/winutil/pull/2310#discussion_r1670306692), nothing else seems out of place (the grammar & wording), so I approve of these changes at the time of writing this review, good commits @MyDrift-user 👍

**Review by Marterich — APPROVED** [review]
Nice Idea and execution :)

**Review by Real-MullaC — CHANGES_REQUESTED** [review]
Check the buttons for Documentation.

**Review by Real-MullaC — APPROVED** [review]

**Review by ChrisTitusTech — APPROVED** [review]

**Pull request merged**